EIPS/eip-4895.md

```diff
@@ -94,21 +130,21 @@ This balance change is unconditional and **MUST** not fail.
 
 This operation has no associated gas costs.
 
-TODO: add logs? this implies receipt structure for the withdrawals and a commitment to the list of receipts
-
 ## Rationale
 
 ### Why not a new transaction type?
 
-This EIP suggests a new type of object -- the "operation" -- as it has special semantics different from other existing types of EVM transactions.
+This EIP suggests a new type of object -- the "withdrawal operation" -- as it has special semantics different from other existing types of EVM transactions.
```

Comment:
I would also mention that it is induced by the system (-> operation) rather than originating from end users (-> transaction).